feat : 정렬기능 구현 , docs : 구현 기능 추가 체크

leejihyeon114 opened the pull request:
1. 회원이름 순 정렬
2. 쿠폰넘버 순 정렬
3. 운동누적시간 순 정렬 ( util 분리 후 수정 예정)
4. 문서 : 운동랭킹 추가 체크

Commit:
docs: 문서변경

1. 쿠폰조회 기능 체크
2. 회원 랭킹 조회(운동시간기준) 체크

Commit:
feat: 회원이름, 쿠폰넘버, 운동누적 정렬

1.회원이름순 정렬
2.쿠폰전체조회 호출
3.회원 누적 운동시간 정렬 ( util로 시간계산 빠지면 추가 수정 예정)

Commit:
feat: 쿠폰 정렬후 전체조회

1.쿠폰 정렬후 전체조회

Commit:
refactor : 쿠폰번호로 정렬 후 전체조회

주석처리된 기존 코드 삭제, 쿠폰번호 정렬기능 추가

Commit:
feat: sort util

1.쿠폰 번호 순 정렬
2.회원 이름 순 정렬
3.운동 누적시간별 정렬

Commit:
style : 쿠폰정렬 리턴타입변경

1. void 에서 collection으로 리턴타입변경

Commit:
style : 이름순정렬 리턴타입 및 출력코드변경

1.void에서 collection으로 리턴타입변경
2.view에서 출력되도록 AdminService에서 출력문 삭제
3.AdminController 유저정보조회 메소드 추가

Commit:
Merge branch 'main' of https://github.com/Inswave-Team5/Health_Management_System

won0845 commented on src/com/healthmanage/service/AdminService.java:
```diff
@@ -114,5 +122,88 @@ public String deleteCoupon(String number) {
+	// 회원 운동시간 누적기준 정렬
+	public void getRank() {
+	      Map<String, Duration> timeForRank = attendanceList.stream()
+	                .filter(a -> !a.getWorkOutTime().isEmpty()) // 운동 시간이 있는 경우만 처리
```
이거 함수가 너무 길어서 분리 해주시면 좋을거 같아여 함수당 기능 하나로 함수에 여러 기능이 있다면 하나씩 만들어서 호출하면 좋을거같아여

won0845 commented on src/com/healthmanage/utils/Sort.java:
```diff
@@ -0,0 +1,53 @@
+                     LinkedHashMap::new
+             ));
+ }
+
```
코드 사이 쓸데없는 공백 지워주시면 좋을 것 같습니다

Commit:
remove : 공백제거

공백 및 주석제거

Commit:
Merge branch 'main' of
https://github.com/Inswave-Team5/Health_Management_System

Commit:
refactor : 운동누적시간순 정렬 time util 적용

1. AdminService : getRank() time util 적용
2. AdminController : view 호출 적용
3. view : showRank() 추가

Commit:
feat : Admin addCoupon() 구현

1. AdminController : addCoupon()구현
2. AdminService : addCoupon() 리턴타입변경
3. AdminView : selectMenu() 추가
4. CouponService : createCoupon() 출력문삭제

Commit:
feat : Admin deleteCoupon() 구현

1. AdminController deleteCoupon()구현
2. CouponService 공백변동

Commit:
Merge branch 'main' of
https://github.com/Inswave-Team5/Health_Management_System
